formats/json-okio/commonMain/src/kotlinx/serialization/json/okio/OkioStreams.kt

```diff
@@ -45,95 +44,79 @@ public fun <T> Json.encodeToSink(
  * @throws [okio.IOException] If an I/O error occurs and sink can't be written to.
  */
 @ExperimentalSerializationApi
-public inline fun <reified T> Json.encodeToSink(
+public inline fun <reified T> Json.encodeToBufferedSink(
     value: T,
-    target: Sink
-): Unit = encodeToSink(serializersModule.serializer(), value, target)
+    target: BufferedSink
+): Unit = encodeToBufferedSink(serializersModule.serializer(), value, target)
 
 
 /**
  * Deserializes JSON from [source] using UTF-8 encoding to a value of type [T] using [deserializer].
  *
- * If [source] is not a [BufferedSource] then called [Source.buffer] for it to create buffered wrapper.
- *
- * Note that this functions expects that exactly one object would be present in the source
```

Comment:
But this function still does `lexer.expectEof()`, why remove this part of documentation?